root/usr/bin/chromium-browser

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
```

Comment:
```suggestion
#!/bin/bash
```
This one doesn't seem like it needs `with-contenv` (it doesn't use any variables being set in the environment) although it probably wouldn't hurt. Either way, no space in the shebang.